src/Infrastructure/Persistence/Doctrine/Repository/Regulation/RegulationOrderRecordRepository.php

```diff
@@ -123,6 +124,50 @@ public function findRegulationOrdersForDatexFormat(): array
         ;
     }
 
+    public function findRegulationOrdersForCifsIncidentFormat(): array
+    {
+        return $this->createQueryBuilder('roc')
+            ->select(
+                'roc.createdAt',
+                'ro.description',
+                'ro.category',
+                'ro.startDate as regulationOrderStartDate',
+                'ro.endDate as regulationOrderEndDate',
+                'loc.address',
+                'ST_X(loc.fromPoint) as fromLongitude',
+                'ST_Y(loc.fromPoint) as fromLatitude',
+                'ST_X(loc.toPoint) as toLongitude',
+                'ST_Y(loc.toPoint) as toLatitude',
+                'm.uuid as measureId',
+                'm.type as measureType',
+                'p.startDateTime as periodStartDateTime',
+                'p.endDateTime as periodEndDateTime',
+                'd.applicableDays',
+                't.startTime',
+                't.endTime',
+            )
+            ->innerJoin('roc.regulationOrder', 'ro')
+            ->innerJoin('roc.organization', 'o')
+            ->innerJoin('ro.locations', 'loc')
+            ->innerJoin('loc.measures', 'm')
+            ->leftJoin('m.periods', 'p')
+            ->leftJoin('p.dailyRange', 'd')
+            ->leftJoin('d.timeSlots', 't')
+            ->where(
```

Comment:
Rajouter une condition pour ne pas prendre en compte celles qui concernent certains véhicules ?

Reply:
Bien vu. Par contre il y a une subtilité en raison de #516, on ne va pas seulement pouvoir faire `WHERE m.vehicleSet IS NULL`. Faut que je réfléchisse...